Anagram.Tests/AnagramUnitTest.cs

```diff
@@ -9,22 +9,22 @@ namespace Anagram.Tests
     public class AnagramUnitTest
     {
         [Fact]
-        public void RecognizesShuffledAlphabetsAsAnagrams()
+        public void ReportsTrueWhenGivenTwoAnagrams()
```

Comment:
Add test cases for other examples in the Wikipedia link, which is given in README.md. Especially the first three. Try to group the examples there, and test each group.